test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -223,6 +214,72 @@ TEST(help_add_test, add_option)
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright copyright information not available."
+                               "================================================================================"
+                               "This program contains SeqAn3 code licensed under the following terms:"
+                               "--------------------------------------------------------------------------------"
+                               + license_string);
+        EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
+                                   expected | ranges::view::remove_if(is_space)));
+    }
+    // Test --copyright with a non-empty short copyright and an empty long copyright.
+    copyright.info.short_copyright = "short copyright\nshortcopyright";
+    {
+        testing::internal::CaptureStdout();
+        EXPECT_THROW(copyright.parse(), parser_interruption);
+        stdout = testing::internal::GetCapturedStdout();
+        expected = std::string("================================================================================"
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright full copyright information not available. Displaying short copyright information instead:"
+                               "short copyright"
+                               "short copyright"
+                               "================================================================================"
+                               "This program contains SeqAn3 code licensed under the following terms:"
+                               "--------------------------------------------------------------------------------"
+                               + license_string);
+        EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
+                                   expected | ranges::view::remove_if(is_space)));
+    }
+    // Test --copyright with a non-empty short copyright and a non-empty long copyright.
+    copyright.info.long_copyright = "long copyright\nlong copyright";
+    {
+        testing::internal::CaptureStdout();
+        EXPECT_THROW(copyright.parse(), parser_interruption);
+        stdout = testing::internal::GetCapturedStdout();
+        expected = std::string("================================================================================"
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright Copyright:"
```

Comment:
actually why do you have `copyright Copyright:` here if in the line above you already have `Copyright information for copyright:"`? isn't that redundant?

Reply:
Good point. Removed it!